response_operations_ui/templates/collection_exercise/ce-dates-section.html

```diff
@@ -1,25 +1,44 @@
 <section class="ce-section ce-events">
     <h2 class="ons-u-fs-r--b ons-u-pt-m">Dates</h2>
+    {% set surveyEditPermission = hasPermission('surveys.edit') %}
 
-    {% set surveyTableData = {
-        "variants": 'compact',
-        "caption": 'Dates for this collection exercise',
-        "hideCaption": true,
-        "ths": [
-            {
-                "value": "Event",
-                "class": "ons-u-vh"
-            },
-            {
-                "value": "Date",
-                "class": "ons-u-vh"
-            },
-            {
-                "value": "Status",
-                "class": "ons-u-vh"
-            }
-        ]
-    } %}
+    {% if surveyEditPermission %}
```

Comment:
This can be refactor like the following:
     ``` {% set surveyTableData = {
          "variants": 'compact',
          "caption": 'Dates for this collection exercise',
          "hideCaption": true,
          "ths": [
              {
                  "value": "Event",
                  "class": "ons-u-vh"
              },
              {
                  "value": "Date",
                  "class": "ons-u-vh"
              },
              {
                  "value": "Status",
                  "class": "ons-u-vh"
              }
          ] if surveyEditPermission else [
              {
                  "value": "Event",
                  "class": "ons-u-vh"
              },
              {
                  "value": "Date",
                  "class": "ons-u-vh"
              }
          ]
      } %}```